README.MD

```diff
@@ -237,7 +246,10 @@ Refer to [DHCP problems](#62-dhcp-problems) if you are having issues with this.
 
 ### 2.4.3 TFTP server
 
-First install required packages:
+TFTP server is installed to enable BBB to get the bootsystem kernel via TFTP.
```

Comment:
TFTP server or (Trivial File Transfer Protocol server) is a simple server from which we can get the bootsystem kernel files.